darwin/future/meta/objects/stage.py

```diff
@@ -17,20 +17,29 @@ class Stage(MetaBase[WFStageCore]):
 
     @property
     def item_ids(self) -> List[UUID]:
-        """_summary_
+        """Item ids attached to the stage
```

Comment:
Actually quite surprised Sphinx didn't moan about this (pre-this change) but maybe we have it excluded in sphinx steps atm